ibc-clients/ics07-tendermint/types/src/client_state.rs

```diff
@@ -235,61 +235,53 @@ impl ClientState {
 impl Protobuf<RawTmClientState> for ClientState {}
 
 impl TryFrom<RawTmClientState> for ClientState {
-    type Error = TendermintClientError;
+    type Error = DecodingError;
 
     fn try_from(raw: RawTmClientState) -> Result<Self, Self::Error> {
         let chain_id = ChainId::from_str(raw.chain_id.as_str())?;
 
         let trust_level = {
-            let trust_level = raw.trust_level.ok_or(DecodingError::MissingRawData {
-                description: "trust level not set".to_string(),
-            })?;
+            let trust_level = raw
+                .trust_level
+                .ok_or(DecodingError::missing_raw_data("missing trust level"))?;
```

Comment:
A quick note: When we use `MissingRawData` and `InvalidRawData`, we end up with an error string like `decoding error: missing raw data: missing trust level` It's a bit unclean and hard to interpret from devex perspective. Could you give this some thought to improve how we display these variants?

Reply:
I cleaned up the description messages to reduce any redundancy, so the message will read as `decoding error: missing raw data: tm client state trust level`. Hopefully that makes things a bit better.

Reply:
That does look better now! Thanks.